README.md

```diff
@@ -124,6 +134,75 @@ you don't have `pip` installed, you can install it with `easy_install pip`.
      Applying users.0001_initial... OK
    ```
 
+### Docker Environment ###
+This documentation assumes you have a working Docker environment set up locally. You can find excellent documentation at [docs.docker.com](http://docs.docker.com).
+
+Note, the documentation assumes your Docker installation defaults to [http://localhost](http://localhost).
+
+Documentation specific for your operating system can be found here:
+
+* [docs.docker.com/linux](https://docs.docker.com/linux/)
+* [docs.docker.com/windows](https://docs.docker.com/windows/)
+* [docs.docker.com/mac](https://docs.docker.com/mac/)
+ 
+#### Linux and OSX ####
+Initial setup is taken care of by the `setup.sh` script in this repository. There is a timing issue that the script corrects; the database needs to up and running before the Django server tries to connect to it.
+
+The dev environment runs on [http://localhost:8000](http://localhost:8000).
+
+##### Initial setup and use: #####
```

Comment:
For reference, @bebef1987 was able to get a local install of docker on Linux working with One and Done.